website/docs/cli/configuring-terraform-cloud/command-line-arguments.html.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: "docs"
+page_title: "Command Line Arguments"
+sidebar_current: "configuring-terraform-cloud"
+description: "Command Line Arguments"
+---
+
+# Command Line Arguments
```

Comment:
> This page is quite short! Do we intend to have more command line arguments in the future? If not, I wonder if there is another place we could put this.

Potentially, yes - that's why I kept it separate, to avoid having to move it later when/if that occurs. However, I'm not super attached to it being separate now so if you have a better place for it, that's cool 👍🏻 I wonder where though; it relates to any command that modifies state, so seems out of place in the `Initialization` page, and feels very random to be in the `Overview` page. 🤔

Reply:
> When would it be necessary/desirable to do this? 

I agree adding something may be good! Though to be honest, I can't come up with an example beyond 'When you don't care about the validity/contents of the remote state and want to overwrite it'; it's a big edge case outside of typical happy paths. 🤔